sdb/commands/count.py

```diff
@@ -23,6 +23,22 @@
 
 
 class Count(sdb.Command):
+    """
+    Return a count of the number of objects in the pipeline
+
+    EXAMPLES
+        Print the number of addresses given
+
+            sdb> addr 0 | count
+            (unsigned long long)1
+            sdb> addr 0 | addr 1 | count
+            (unsigned long long)2
+
+        Print the number of dbufs
+
+            sdb> dbuf | count
+            (unsigned long long)19
```

Comment:
That's a good example but `dbufs` is a very generic term for someone who is not familiar with ZFS.
Maybe change the description to say `Print the number of ZFS dbufs`.

Other examples that are more generic for Linux folks are:
```
    Print the number of root slab caches in the system

        sdb> slabs | count
        (unsigned long long)136

    Count the number of level 3 log statements in the kernel log buffer.

        sdb> dmesg | filter obj.level == 3 | count
        (unsigned long long)24
```